ethclient/gethclient/gethclient.go

```diff
@@ -265,3 +287,52 @@ func (a OverrideAccount) MarshalJSON() ([]byte, error) {
 	}
 	return json.Marshal(output)
 }
+
+// BlockOverrides specifies the  set of header fields to override.
+type BlockOverrides struct {
```

Comment:
Perhaps add a comment that this struct is a direct "replica" of the `BlockOverrides` defined in `interna/ethapi/api.go`. 
Would it make sense to have it defined in only one place (thus remove it from internal)?

Reply:
We kinda need both. The API needs fields with suitable types for input (i.e. `hexutil`), and gethclient needs standard types which will then be serialized to be sent to the API. That's the reason why I copied the structure and changed the types.